docusaurus/docs/what-is-included/available-scripts.md

```diff
@@ -0,0 +1,104 @@
+---
```

Comment:
We should make note that the iOS binary to upload to TestFlight, i.e., the iOS App Store Package (IPA) file, can only be created through Xcode at the moment.